### DISCUSSION TITLE
How to Exclude 2fa Authentication for specific Users

### DISCUSSION THREAD

**Discussion by mukitaro** [gravitational/teleport#13189] [Q&A] [answered]
Hi,
Is it possible to exclude 2fa authentication for specific users or roles?
I want to automate the process of adding Node to Teleport Cluster etc.

**Comment by webvictim** [comment, marked as answer]
This isn't currently possible - the second factor setting applies to the entire cluster.

You can automate adding nodes to a cluster using a dynamic token - there is no user account required to do this: https://goteleport.com/docs/server-access/getting-started/#step-24-add-a-node-to-the-cluster

**Reply by mukitaro** [reply]
Hi webvictim

Thank you for providing an answer to my question.
I understood it.
